server/settings/publish_plugins.py

```diff
@@ -11,6 +11,11 @@ class ValidatePluginModel(BaseSettingsModel):
     active: bool = SettingsField(True, title="Active")
 
 
+class MiniValidatePluginModel(BaseSettingsModel):
+    
```

Comment:
```suggestion
```

I also think a better name is `EnabledStateModel` instead of `MiniValidatePluginModel`

Reply:
Thank you
https://github.com/ynput/ayon-traypublisher/pull/51/commits/fc5dcbb47bea50492f3b0ad341a07990fc9c3be3